### PR TITLE
Madeleine update tile

### PR DESCRIPTION
**Pull request by GeOdin**

### CODE REVIEW

File: src/components/AppIcon.css [hunk 1]
```diff
@@ -23,13 +21,16 @@
...
+  width: 100%;
   padding-left: 10px;
   padding-right: 10px;
-  font-size: 18px;
-  width: 100%;
-  text-align: center;
   color: #333;
+  font-family: Montserrat;
+  font-size: 17px;
+  font-color: #FFFFFF;
```
Comment (hoanphungt) [inline src/components/AppIcon.css]: What do you want to change with font-color? if it is to change the color of the font then color is the one you need and it's already set at line 28.

File: src/components/AppIcon.js [hunk 2]
```diff
@@ -6,12 +6,10 @@ import { withRouter } from "react-router-dom";
 
 class AppIcon extends Component {
   render() {
-    const { src, title, subTitle, handleClick } = this.props;
+    const { title, bgImage, handleClick } = this.props;
     return (
-      <div className={styles.AppIcon} onClick={handleClick}>
-        <img src={src} alt={title} className={styles.Img} />
+      <div className={styles.AppIcon} style={{backgroundImage: `url(${bgImage})`}} onClick={handleClick}>
```
Comment (hoanphungt) [inline src/components/AppIcon.js]: Nice that you use backgroundImage here for the whole tile.

File: src/home/App.js [hunk 3]
```diff
@@ -52,7 +52,7 @@ class App extends Component {
             defaultMessage="Data Management"
           />
         ),
-        bgImage: templateIcon
+        bgImage: templateTileImage
```
Comment (hoanphungt) [inline src/home/App.js]: I find this name is quite misleading, it should be something like dataManagementTileImage. You also have a templatesTileImage in Alarms.js

File: src/components/AppIcon.css (hunk 1 above)
Comment (GeOdin) [reply src/components/AppIcon.css]: Good idea! It is indeed no longer necessary.

File: src/home/App.js (hunk 3 above)
Comment (GeOdin) [reply src/home/App.js]: Thank you! :) Forgot to rename it.

Comment (GeOdin) [reply src/home/App.js]: You made my day. :)

**Comment by GeOdin** [comment]
Thank you!

For me de font is actually the same as for the design of Lirry.
Could you pull the madeleine-redesign-management-client branch again and update it?
(`git pull origin madeleine-redesign-management-client` when on the branch)

**Pull request merged**